lib/bolt/config/options.rb

```diff
@@ -0,0 +1,320 @@
+      #            should use the TrueClass type.
+      OPTIONS = {
+        "apply_settings" => {
+          desc: "A map of Puppet settings to use when applying Puppet code using the `apply` "\
+                "plan function or the `bolt apply` command.",
+          type: Hash
+        },
+        "color" => {
+          desc: "Whether to use colored output when printing messages to the console.",
+          type: TrueClass,
+          exmp: false,
+          def: true
+        },
+        "compile-concurrency" => {
+          desc: "The maximum number of simultaneous manifest block compiles.",
+          type: Integer,
+          exmp: 4,
+          def: "Number of cores."
+        },
+        "concurrency" => {
+          desc: "The number of threads to use when executing on remote targets.",
+          type: Integer,
+          exmp: 50,
+          def: "100 or 1/7 the ulimit, whichever is lower."
+        },
+        "format" => {
+          desc: "The format to use when printing results. Options are `human`, `json`, and `rainbow`.",
+          type: String,
+          exmp: "json",
+          def:  "human"
+        },
+        "hiera-config" => {
+          desc: "The path to your Hiera config.",
+          type: String,
+          def:  "boltdir/hiera.yaml",
+          exmp: "~/.puppetlabs/bolt/hiera.yaml"
+        },
+        "inventory-config" => {
+          desc: "A map of default configuration options for the inventory. This includes options "\
+                "for setting the default transport to use when connecting to targets, as well as "\
+                "options for configuring the default behavior of each transport.",
+          type: Hash
+        },
+        "inventoryfile" => {
+          desc: "The path to a structured data inventory file used to refer to groups of targets on the command "\
+                "line and from plans. Read more about using inventory files in [Inventory "\
+                "files](inventory_file_v2.md).",
+          type: String,
+          def:  "boltdir/inventory.yaml",
+          exmp: "~/.puppetlabs/bolt/inventory.yaml"
+        },
+        "log" => {
+          desc: "A map of configuration for the logfile output. Configuration can be set for "\
+                "`console` and individual log files, such as `~/.puppetlabs/bolt/debug.log`. "\
+                "Each key in the map is the logfile output to configure, with the corresponding "\
+                "value configuring the logfile output.",
+          type: Hash,
+          exmp: { "console" => { "level" => "info" },
+                  "~/logs/debug.log" => { "append" => false, "level" => "debug" } }
+        },
+        "modulepath" => {
+          desc: "An array of directories that Bolt loads content such as plans and tasks from. Read more "\
+                "about modules in [Module structure](module_structure.md).",
+          type: [Array, String],
+          def:  ["boltdir/modules", "boltdir/site-modules", "boltdir/site"],
```

Comment:
I don't feel particularly strongly, but should we change this to project?

Reply:
I actually had that before I opened the PR. I was convinced in review we would want to change it to boltdir instead haha. I'll update.

Reply:
Heh, damned if you do damned if you don't. @hestonhoffman Might have a more informed opinion.